src/core/abr/representation_chooser.ts

```diff
@@ -280,6 +296,31 @@ export default class RepresentationChooser {
   private estimator : BandwidthEstimator;
   private _initialBitrate : number;
 
+  /**
+   * Information about frames (total decoded and dropped)
+   * Emitted at each clock event.
+   */
+  private lastDroppedFrames: number = 0;
+  private lastTotalFrames: number = 0;
+  private currentDroppedFrames: number = 0;
+  private currentTotalFrames: number = 0;
+
+  /**
+   * Maximum bitrate for which stream can be easily decoded.
+   * It is the last representation bitrate before the
+   * representation that causes player to have decode issues.
+   */
+  private _decodableBitrate: number = Infinity;
```

Comment:
``undefined`` would be a better default value no?

Reply:
I prefer to set this to ``Infinity``, while setting it to ``undefined`` and filtering it before making a ``Math.min```operation on it later.
Decodable Bitrate (which may be redefined as max decodable bitrate ?), can be considered as litteraly infinite while it hasn't been set to a finite max value.